build_scripts_examples/Ansible_Tower/liquibase_playbook.yml

```diff
@@ -0,0 +1,91 @@
+###
+### This sample pipeline is intended as a starting point for integration between Liquibase Pro and Ansible
+### Review and update prior to use
+###
+### Liquibase sales: https://www.liquibase.com/contact
+### Liquibase support: https://support.liquibase.com/knowledge
+### 
+
+###
+### Prior to running the following files must exist in repository:
+###
+### 1. liquibase.flowfile.yaml
+###    Liquibase flow file
+###    https://docs.liquibase.com/commands/flow/flow.html
+###
+### 2. liquibase.checks-settings.conf
+###    Liquibase quality checks configuration file
+###    https://docs.liquibase.com/commands/quality-checks/home.html
+###
+### 3. Snapshot_DEV.json
+###    Initial snapshot file for target database (will be updated as part of the Liquibase flow)
+###    Run: liquibase snapshot --outputfile="Snapshot_DEV.json" --snapshotformat=json
+###    https://docs.liquibase.com/commands/inspection/snapshot.html
+###
+### 4. changelog.main.xml
+###    Liquibase changelog file
+###    https://docs.liquibase.com/concepts/changelogs/home.html
+###
+
+###
+### Ansible documentation: https://docs.ansible.com/
+###
+
+---
+
+- name: Running Liquibase
+  hosts: localhost
+  vars:
+    LB_TMP_DIR: /tmp/Toolbox
+  tasks:
+###
+### Checkout repository
+### This example uses a personal access token, adjust as required
+###
+    - name: Checking out repository
+      ansible.builtin.git:
+        repo: "https://{{ GITHUB_TOKEN }}@github.com/jbennett-liquibase/Toolbox.git"
+        dest: "{{ LB_TMP_DIR }}"
```

Comment:
Does dest: create a /tmp/Toolbox/Toolbox or does it do more of a git clone URL outputdir = LB_TMP_DIR?

Reply:
It creates the LB_TMP_DIR directory and then does a clone into it.